sky/skylet/job_lib.py

```diff
@@ -125,6 +135,73 @@ def colored_str(self):
         return f'{color}{self.value}{colorama.Style.RESET_ALL}'
 
 
+# Only update status of the jobs after this many seconds of job submission,
```

Comment:
Reorder the L89 and L103 as now `SETTING_UP` will come after the `PENDING` status